keyboards/sawnsprojects/okayu/info.json

```diff
@@ -0,0 +1,337 @@
+{
+    "keyboard_name": "Okayu",
+    "manufacturer": "Salane",
+    "url": "",
+    "maintainer": "Mai The San",
+    "usb": {
+        "vid": "0x5350",
+        "pid": "0x00A1",
+        "device_version": "1.0.0",
+        "force_nkro": true
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": false,
+        "nkro": true,
+        "rgblight": true
+    },
+    "community_layouts": [
+        "60_ansi",
+        "60_ansi_split_bs_rshift",
+        "60_ansi_tsangan"
+    ],
+    "matrix_pins": {
+        "cols": ["A9", "A0", "A8", "C13", "A1", "A2", "B11", "B10", "B1", "B0", "A7", "A6", "A5", "A4"],
+        "rows": ["B3", "A15", "A3", "A10", "B14"]
+    },
+    "diode_direction": "COL2ROW",
+    "indicators": {
+        "caps_lock": "B12",
+        "on_state": 0
+    },
+    "ws2812": {
+        "pin": "B15",
+        "driver": "spi"
+    },
+        "rgblight": {
+        "led_count": 24,
+        "sleep": true,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "layouts": {
```

Comment:
could the following QMK Community Layouts be added?:
- `60_tsangan_hhkb`
- `60_hhkb`
- `60_iso`
- `60_iso_split_bs_rshift`
- `60_iso_tsangan`
or a `matrix_disgram.md` provided